### DISCUSSION TITLE
How do I use "ferium profile configure" to configure a profile with a specific name?

### DISCUSSION THREAD

**Discussion by TechPro424** [gorilla-devs/ferium#123] [Q&A] [answered]
How do I use `ferium profile configure` to configure a profile with a specific name? Is it `ferium profile configure <name>`?

**Comment by theRookieCoder** [comment, marked as answer]
Running only `ferium profile configure` will show a UI where you can change settings. You can also change settings using options, e.g. for changing the name you will have to run `ferium profile configure --name <new_name>`

**Reply by TechPro424** [reply]
No, what I meant is, I have a profile created with a specific name, e.g: x
I also have another profile y
How do I configure the settings of profile x using cli flags?

**Reply by theRookieCoder** [reply]
Ohh, well you can't do that. You have to switch the profile first using `ferium profile switch <profile_name>`

**Reply by TechPro424** [reply]
So, I have to run `ferium profile switch x` and then change the settings?
